api/services/redis/index.js

```diff
@@ -0,0 +1,23 @@
+const config = require('config');
+const redis = require('redis');
+const { promisify } = require('util');
+
+class RedisClient {
+  constructor(app) {
+    this._app = app;
+    const client = redis.createClient();
+    client.on('error', function (error) {
+      console.error(error);
+    });
+
+    const getAsync = promisify(client.get).bind(client);
+    client.get = getAsync;
+    this._client = client;
+  }
+
+  get client() {
```

Comment:
I mentioned the change in one of the above comments wherein we'll be creating wrapper methods like set for our use case. The reasoning behind that is to have an interface in our own service rather than relying on the interface of the library that we are using.